doc/util.rst

```diff
@@ -72,3 +72,11 @@
     Return the root mean square of signal ``s`` given the FFT transform
     ``f = fft(s)``. Equivalent to ``rms(ifft(f))``.
 
+.. function:: meanfreq(f)
```

Comment:
Function argument should be `x` here to match the description. Second function argument is missing.